providers/stores/mongo.js

```diff
@@ -63,6 +60,29 @@ class MongoStore {
     return definition
   }
 
+  /**
+   * Query and return the objects based on the query
+   *
+   * @param {object} query - The filters and sorts for the request
+   * @returns The data and continuationToken if there is more results
+   */
+  async find(query, continuationToken = '') {
+    const pageSize = 50 // no option for page size, just next tokens
+    const filter = this._buildQuery(query, continuationToken)
+    const sort = this._buildSort(query)
+    const cursor = await this.collection.find(filter, {
+      projection: { _id: 0, files: 0 },
+      sort: sort,
+      limit: pageSize
+    })
+    const data = await cursor.toArray()
+    continuationToken = this._getContinuationToken(pageSize, data)
+    data.forEach(def => {
+      delete def._mongo
```

Comment:
can we use the `projection` here to remove `_mongo` or does that mess up the query?